scripts/vm/hypervisor/kvm/kvmheartbeat.sh

```diff
@@ -156,7 +156,8 @@ then
 elif [ "$cflag" == "1" ]
 then
   /usr/bin/logger -t heartbeat "kvmheartbeat.sh rebooted system because it was unable to write the heartbeat to the storage."
-  sync
+  sync &
```

Comment:
what if sync takes more than 5 seconds?
Why are we sync-ing at all if we don't want to block before rebooting?